### DISCUSSION TITLE
Handle multiple repository and multiple clusters in flux CD v2 without using helm repository

### DISCUSSION THREAD

**Discussion by meghadimpu** [fluxcd/flux2#1781] [Q&A]
How to handle multiple repository and multiple clusters in flux CD v2 without using helm repository

**Comment by kingdonb** [comment]
Can you please clarify your question?

You can use multiple git repository objects to sync to multiple clusters, using a Flux Kustomization for each cluster. The Kustomization can each reference a kubeconfig which gives the needed access to the remote cluster.

Please take a look at this doc and see if it helps: https://fluxcd.io/docs/components/kustomize/kustomization/#remote-clusters--cluster-api

Thanks for using Flux!
